### ISSUE TITLE
Star ranking is not accessible via keyboard

### ISSUE THREAD

**Issue by emilinecsa** [asc-csa/ckanext-asc-csa#221]
_---- Reported by Communications ----_

Star ranking is not accessible via keyboard, which causes an accessibility problem. Ideally, we would have a solution using radio buttons and CSS to replace them with stars. See this example:
[https://www.cssscript.com/accessible-star-rating-system-pure-css/](https://www.cssscript.com/accessible-star-rating-system-pure-css/)

**Comment by emilinecsa** [comment]
We deployed the latest version of the application and this error:
`No such file or directory: '/usr/lib/ckan/default/src/ckanext-feedback/ckanext/feedback/assets/feedback.css'`

By looking in the code, this [HTML template](https://github.com/asc-csa/ckanext-asc-csa/blob/cleanups/ckanext/csa/templates/feedback/snippets/rating_widget.html) has extra code (line 40) to load a new CSS file: `{% asset "ckanext_feedback/feedback_css" %}`. None of the feedback projects contain `feedback.css`. Is it possible that you just forgot to add it to source control?

**Comment by emilinecsa** [comment]
This problem is now fixed. Tested on May 22, 2026.

<img width="270" height="152" alt="Image" src="https://github.com/user-attachments/assets/e23d6898-8a31-4903-a1ca-4819235d306e" />
